docs/getting-started-with-cassandra.md

```diff
@@ -0,0 +1,37 @@
+## Getting Started with Scalar DB on Cassandra
```

Comment:
```suggestion
# Getting Started with Scalar DB on Cassandra
```
for consistency

Reply:
Thank you for your suggestion.
The document updated accordingly.